docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -121,26 +127,32 @@ endif::[]
 
 Before you use {product-connectors}, you must complete the following prerequisites: 
 
-* Determine which {openshift} environment to use for deploying your {product-connectors} instances.
+* Determine which {openshift} environment to use for your _{connectors} namespace_. The {connectors} namespace is where your {product-connectors} instances will be deployed.
 
 * Configure {product-long-kafka} for use with {product-connectors}.
 
-*Determining which {openshift} environment to use for deploying your {connectors} instances*
+*Determining which {openshift} environment to use for your {connectors} namespace*
 
-For Service Preview, you have two choices:
+You have three choices:
 
 * *The hosted preview environment*
 
 ** The {connectors} instances are hosted on a multitenant {osd-name-short} cluster that is owned by Red Hat.
 ** You can create four {connectors} instances at a time.
-** The preview environment applies 48-hour expiration windows, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/8190dc9e-249c-4207-bd69-096e5dd5bc64[Red Hat {openshift} {connectors} Service Preview guidelines^].
+** The preview environment applies 48-hour expiration windows, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/8190dc9e-249c-4207-bd69-096e5dd5bc64[Red Hat {openshift} {connectors} Preview guidelines^].
 
-* *Your own trial environment*
+* *Your own trial {osd-name-short} environment*
 
 ** You have access to your own {osd-name-short} trial environment.
```

Comment:
Change "your own" to "an", for consistency with the body text for ROSA. Or perhaps change the ROSA entry to "your own", if that distinction is important. Or leave as is, if both are correctly different.

```suggestion
** You have access to an {osd-name-short} Trial environment.
```